src/components/Shared/GridToolbar.js

```diff
@@ -18,6 +18,7 @@ const GridToolbar = ({
   initialLoad,
   onAdd,
   openRPB,
+  onTree,
   disableRPB = false,
```

Comment:
Let's add change this into actions.
`[{
image: '/images/buttonsIcons/tree.png',
onClick(){}
}]`